helm/efk-stack-app/charts/opendistro-es/templates/elasticsearch/es-client-ingress.yaml

```diff
@@ -13,10 +13,14 @@
 
 # @formatter:off
 {{- if and .Values.elasticsearch.client.ingress.enabled .Values.elasticsearch.client.enabled }}
-{{ $fullName := printf "%s-%s"  (include "opendistro-es.fullname" .) "client-service" }}
+{{ $fullName := printf "%s-%s"  (include "opendistro-es.fullname" .) "client-ingress" }}
```

Comment:
Did you test this name change? Looks like this is the service name selector, but also used for the ingress name.

Looks like this service is targeted https://github.com/giantswarm/efk-stack-app/blob/master/helm/efk-stack-app/charts/opendistro-es/templates/elasticsearch/es-service.yaml#L24

Reply:
Do not change the resource name here, please. This makes it difficult to sync with upstream, and we already fix many issues here with incorrect resources naming.